index.rst

```diff
@@ -10,20 +10,849 @@
+Throughout this report, this summit-dedicated payload is referred to as Rapid Analysis, whose instantiation would ensure that on-the-fly diagnostic information is available to observers.
+Running the Rapid Analysis payload on the summit, which is shown to be feasible in :ref:`Antu at Summit <antu_at_summit>`, would increase the robustness from network outages, and may also provide a faster response depending on the desired reductions.
+
+Responses to Charge Questions and Deliverables
+==============================================
+
+The use-cases referenced throughout the document are all found on the `dedicated confluence page <https://confluence.lsstcorp.org/display/LSSTCOM/Use-Cases>`_ and are further described below.
+
+.. _Deliverable 1:
+
+Deliverable 1: Use-Cases
+------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+   1. (`SITCOM-174`_, `SITCOM-173`_, `SITCOM-180`_) A series of use-cases where image data analysis is required on short timescales.
+      A reduced set of use-cases should be created as a regular reference throughout the charge.
+      A set of required turn-around time(s) should be defined and assigned to each case where applicable.
+
+      - Use-cases should be complete, including which inputs are required and from where they will originate (e.g. SAL Script, EFD, LFA, external source), desired manipulations, logic-based operations/calculations, and if/how the desired artifacts are presented to the user (e.g. display images and/or graphs).
+
+
+Numerous use cases were developed to capture the needed functionalities and assist in developing a common understanding of what is expected in each scenario.
+Each of the use cases follow a standardized `template <https://confluence.lsstcorp.org/display/LSSTCOM/On-the-fly+Analysis+Use-Case+Template>`_ which differs slightly from that which was used in the first FAFF charge.
+
+The remaining use-cases for FAFF2 can be found on the FAFF use-cases page `on confluence <https://confluence.lsstcorp.org/display/LSSTCOM/Use-Cases>`_ and are referenced throughout the remainder of this report.
+
+Daytime Calibration
+^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+   This section is not yet completed and only provides a status update.
+
+
+During the course of the working group, the example of daytime calibration was raised repeatedly, specifically in regards to how calibration data products are generated and what is expected of the observing specialist.
+The aspect pertaining specifically to the FAFF charge is what the observer is required to look at during the process, including both images and/or alarms.
+The details of how Daytime Calibration is performed is being documented in `DMTN-222 <https://DMTN-222.lsst.io>`_ and will not be repeated as a new FAFF use-case.
+
+In short, a SAL script is launched by the observer to acquire a daytime set of calibrations.
```

Comment:
SAL Script

Reply:
I don't understand this comment? I don't see an obvious issue in the sentence either...

Reply:
The "s" in "script" should be capitalized.. We can them "SAL Script"....